interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -592,12 +572,56 @@ const Page = () => {
               },
             },
           });
+
+          await new Promise((resolve) => setTimeout(resolve, 3000));
+
+          const result = await getPackagesForSim({
+            variables: {
+              data: {
+                sim_id: plan.simId,
+              },
+            },
+          });
+
+          const packages = result?.data?.getPackagesForSim?.packages || [];
```

Comment:
Unnecessary `timeout` here. API's are async. If you want to do something after api call response do that in `onComplete` OR `onError` callbacks.